### ISSUE TITLE
Helper error on fresh clone

### ISSUE THREAD

**Issue by greenmanspirit** [hugodias/cakegallery#9]
I just cloned this repo into my project and I get them Warning immediately.

Warning (512): Method GalleryHelper::new_gallery_button does not exist [CORE/Cake/View/Helper.php, line 192]

I checked the file here on github as well and that function does not exist.

**Comment by greenmanspirit** [comment]
I also noticed that there is no code in general for adding albums. Why is this?

**Comment by ptica** [comment]
I believe that a
```
# in Plugin/Gallery/View/Gallery/index.ctp
echo $this->Gallery->link(null, null,
       array('class' => 'btn btn-primary pull-right', 'style' => 'margin-top: 10px')
);
```
is now used instead -- a link to upload photos, as used for example here:
https://github.com/hugodias/cakegallery/blob/master/View/Gallery/index.ctp#L11

**Comment by hugodias** [comment]
@ptica fixed that in the master branch on #10 . Check that out @greenmanspirit
